src/StaticFAQ/ArticleFeedback/ScreenCommentLimit.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+
```

Comment:
this component is suspiciously similar to ScreenError, doesn't it make sense to have just one?